boileroom/models/minifold/core.py

```
@@ -0,0 +1,442 @@
+        "device": "cuda:0",
+        "model_size": "48L",  # "48L" or "12L"
+        "num_recycling": 3,
+        "compile": False,
+        "kernels": False,
+        "cache_dir": None,  # defaults to MODEL_DIR/minifold
+        "glycine_linker": "",  # empty = direct concat (monomer default)
+        "include_fields": None,  # Optional[List[str]] - controls which fields to include in output
+    }
+    STATIC_CONFIG_KEYS = {"device", "model_size", "cache_dir", "compile", "kernels"}
+
+    def __init__(self, config: dict = {}) -> None:
```

Ruff (0.15.1)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)